README.md

```diff
@@ -28,11 +29,21 @@ Install the [plugin](https://plugins.jenkins.io/mabl-integration) into your Jenk
 
 ### Requirements
 
--   Minimum Jenkins version *1.580.1*
+-   Minimum Jenkins version: *2.121.3*
+-   Minimum Java runtime version: *8*
 -   mabl API key
     -   See [integration
         docs](https://help.mabl.com/v1.0/docs/integrating-mabl-with-your-cicd-workflow#section-the-mabl-deployment-events-api)
         for details  
+
+### Create a Jenkins credential for the mabl API key
+
+-   Add a new credential accessible to the Jenkins job using the Secret text
```

Comment:
Does it have to be system global credential? There are a couple of scopes there when plugin is installed, perhaps we can clarify that.

Reply:
Sounds good. It has to be global scoped credential, but the credential can live in the non-global domain. I'll clarify.

Reply:
👍